packages/api/src/libs/queries/lsp.ts

```diff
@@ -0,0 +1,85 @@
+import assert from "assert";
```

Comment:
this is a little utility library which is mostly copied from queries/emp but tailored to the LSP. mainly just joins state across tables, and sums tvls for now.  used as a utility in the actions (api) layer and some services